Makefile

```diff
@@ -355,8 +359,10 @@ check: protoc ## Dockerized version of make test
 	$(DOCKER_GO) "make test"
 
 .PHONY: container-build
-container-build: ## Build containers
-	make docker-build bundle bundle-build
+container-build:docker-build bundle bundle-build ## Build containers
+
+.PHONY: container-build-check
+container-build-check:docker-build-check bundle bundle-build ## Build containers
```

Comment:
Two small NITs:
1. I don't think we need to call for `bundle` twice here. First on this line, and then again in [`bundle-build`](https://github.com/medik8s/self-node-remediation/blob/main/Makefile#L276-L278)
2. A space is missing right after the target name :)